static/base/js/client.js

```diff
@@ -113,7 +113,7 @@
             };
             this.request('GET', '/accounts/analytics/', params, {}, callback);
         },
-        getBrowsePlaylist: function(emotion, jitter, limit, genre, context, callback) {
+        getBrowsePlaylist: function(emotion, jitter, limit, genre, context, returnLast, callback) {
```

Comment:
We can remove the `returnLast` param from this function; the endpoint no longer supports it